.github/workflows/examples.yaml

```diff
@@ -258,7 +258,7 @@ jobs:
           SLACK_WEBHOOK: ${{ secrets.ML_TEAM_EXAMPLES_WEBHOOK }}
           SLACK_USERNAME: "Example Runs Root Content Populator"
           SLACK_ICON_EMOJI: ":telescope:"
-          SLACK_COLOR: ${{ ((needs.mltc-example.result == 'success') && (needs.ner-example.result == 'success') && (needs.tc-example.result == 'success') && (needs.ic-example.result == 'success') && (needs.od-example.result == 'success') && (needs.semseg-example.result == 'success') && 'success') || 'failure' }}
+          SLACK_COLOR: ${{ ((needs.mltc-example.result == 'success') && (needs.ner-example.result == 'success') && (needs.tc-example.result == 'success') && (needs.ic-example-HF.result == 'success') && (needs.od-example.result == 'success') && (needs.semseg-example.result == 'success') && 'success') || 'failure' }}
```

Comment:
same here, should be changed back. if there's an inference example run we can add a separate job in the dag.